editors/code/src/Commands/createCHANGELOG.ts

```diff
@@ -2,31 +2,34 @@ import { workspace as Workspace, window, extensions } from "vscode";
 import { existsSync, copyFileSync } from "fs";
 import { join } from "path";
 
-export function run(rootpath?: string) {
-  // get workspace folder
+export function run(rootpath?: string): void {
```

Comment:
The functions return non zero values when they error out. This is used in automated tests.